### PR TITLE
Fix codemirror scrollbars, misc improvements

### LINKED ISSUE #1286: Use codemirror scrollbars
The new playground editor should use codemirror scrollbars. The embeds already use them, so this should be a quick upgrade.

### PR DESCRIPTION
**Pull request by johnpryan**
- improve scrollbar + copy button padding
- fix busylight alignment
- HTML formatting

closes #1286

### CODE REVIEW

File: lib/new_playground.dart [hunk 1]
```diff
@@ -46,6 +47,25 @@ import 'src/ga.dart';
 import 'src/util.dart';
 import 'util/detect_flutter.dart';
 
+var codeMirrorOptions = {
```
Comment (RedBrogdon) [inline lib/new_playground.dart]: nit: can this be `const`?

File: web/experimental/styles.scss [hunk 2]
```diff
@@ -196,6 +197,8 @@ body {
   z-index: 10;
   font-size: 18px;
   color: $mdc-theme-primary;
+  margin-right: 1em;
```
Comment (RedBrogdon) [inline web/experimental/styles.scss]: Almost all the other margin/padding measurements in here are in raw pixels. Is this one in ems for a particular reason?

Comment (johnpryan) [reply web/experimental/styles.scss]: This is a holdover from the old playground styles. changed to 8px.

File: lib/new_playground.dart (hunk 1 above)
Comment (johnpryan) [reply lib/new_playground.dart]: yep, fixed.

**Pull request merged**